memory/dma_memtest.py.data/dma_memtest.yaml

```diff
@@ -0,0 +1,7 @@
+setup:
+    tarball_base: !mux
```

Comment:
This is parametrization, so you can drop the `!mux` tag, unless you are thinking in run the test for more than one linux kernel.